src/cli/commands/mod.rs

```diff
@@ -0,0 +1,75 @@
+//! This mod contains the different command line implementations.
+
+mod server;
+
+use crate::cli::CommandLineHandler;
+use clap::{Parser, Subcommand};
+use std::fmt::Debug;
+
+/// The `cli` method exposed to handle all the cli commands, ideally from main.
+///
+/// Consider using macro_rules! to handle the registration of new CLI handlers.
```

Comment:
This is no longer the case, right? We don't use macros.

Reply:
removed, resolving this